sites/public/styles/overrides.scss

```diff
@@ -31,6 +31,10 @@ h1.form-card__header_title {
   text-transform: capitalize;
 }
 
+li.progress-nav__item {
```

Comment:
Please add a comment explaining why this is the case.

Reply:
Done, thanks!